sklearn/utils/_openmp_helpers.pyx

```diff
@@ -0,0 +1,3 @@
+def _openmp_supported():
+    """Determines if scikit-learn has been built with OpenMP support"""
+    return SKLEARN_OPENMP_SUPPORTED
```

Comment:
I'm confused by this

How can this be both a runtime constant like here, and used as a pre-processor macro as  described in e.g. `doc/developers/performance.rst`?